src/sponge/pad.rs

```diff
@@ -1,16 +1,17 @@
-use curve25519_dalek::scalar::Scalar;
+//! Padding support for sponge hash
+//!
 
```

Comment:
Maybe we could be more descriptive about what it does, at least how the `pad_value` and `eom` are used.

Reply:
Not targetting to have a good documentation atm.

I just put this so I don't get errors due to the flag `#[deny(missing_docs)]`.
I was leaving docs for the end once #11 gets merged.